pilot/pkg/networking/plugin/authz/rbac_test.go

```diff
@@ -496,7 +496,7 @@ func TestConvertRbacRulesToFilterConfig(t *testing.T) {
 					},
 					{
 						Properties: map[string]string{
-							"source.principal": "user",
+							attrSrcPrincipal: "user",
```

Comment:
Good point, but I think we should use the variable `attrSrcPrincipal` for a couple of reasons:
1. If someone accidentally changes the value of `attrSrcPrincipal`, then when the user defines a value different than `attrSrcPrincipal` (since it has changed), the user-provided value won't match with `attrSrcPrincipal` and will either be ignored or handled as a default value. So when the test is run, the `expect` and `actual` won't match -> Test will fail.
2. Using the constant string rather than `attrSrcPrincipal` means we will not only have to change where `attrSrcPrincipal` is used, but other keys in rbac.go as well (lots of places in checked in code already use the key variables). In the future, if we purposely want to change a key value, we will need to change it in the test as well, which can be tiresome. 
Lkm what you think :)

Reply:
I see, that makes sense, let's keep it.